absl/debugging/CMakeLists.txt

```diff
@@ -62,6 +62,9 @@ absl_cc_test(
     "symbolize_test.cc"
   COPTS
     ${ABSL_TEST_COPTS}
+    $<$<BOOL:${MSVC}>:-Z7>
```

Comment:
From the internal code review: Is there a reason to use `-Z7` and `-DEBUG` here over `/Z7` and `/DEBUG` as you had it in the Bazel file?

Reply:
For some reason using `/` instead of `-` in CMake generator expression causes weird CMake error, so I just use `-`.